cdm/misc/src/main/java/ucar/nc2/geotiff/GeotiffWriter.java

```diff
@@ -213,17 +331,55 @@ private void writeMetadata(boolean greyScale, double xStart, double yStart, doub
       geotiff.addTag(new IFDEntry(Tag.XResolution, FieldType.RATIONAL).setValue(1, 1));
       geotiff.addTag(new IFDEntry(Tag.YResolution, FieldType.RATIONAL).setValue(1, 1));
       geotiff.addTag(new IFDEntry(Tag.ResolutionUnit, FieldType.SHORT).setValue(1));
-
+      // black is zero (value used in GeotiffWriter)
+      geotiff.addTag(new IFDEntry(Tag.PhotometricInterpretation, FieldType.SHORT).setValue(1));
     } else {
+      if (colorTable != null && colorTable.length > 0) {
+        // standard tags for Palette-color images ( see TIFF spec, section 5)
+        geotiff.addTag(new IFDEntry(Tag.PhotometricInterpretation, FieldType.SHORT).setValue(3));
+        geotiff.addTag(new IFDEntry(Tag.ColorMap, FieldType.SHORT, colorTable.length).setValue(colorTable));
```

Comment:
Just realized that I need to have this in the greyscale mode, too, if I want to be consistent with what I documented. I should also add a unit test proving out that this works.

Reply:
Just fixed this, and I think I vastly simplified `writeMetadata()`, too.